agent/handshake/agency.go

```diff
@@ -113,6 +115,18 @@ func AnchorAgent(agentName, seed string) (agent *cloud.Agent, caDid core.DID, er
 	caDid = try.To1(agent.NewDID(method.TypeSov, ""))
 	try.To(enclave.SetKeysDID(key, caDid.Did()))
 
+	glog.V(2).Infof("Creating a master secret into worker's wallet (%s)", caDid.Did())
+	masterSec, err := enclave.NewWalletMasterSecret(caDid.Did())
```

Comment:
maybe use `try.To1` now when we have `defer err2.Handle` in this same function? Or even `assert.NoError` to underline that this would be an error we cannot recover?